src/Checkbox.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import cx from 'classnames';
+import Flex from './Flex';
+import FlexItem from './FlexItem';
+
+/**
+ * @module Checkbox
+ */
+class Checkbox extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			checked: props.checked || false
+		};
+		this.onChange = this.onChange.bind(this);
+	}
+
+	onChange(e) {
+		this.setState({ checked: e.target.checked });
+	}
+
+	render() {
+		const {
+			checked, // eslint-disable-line no-unused-vars
+			children,
+			className,
+			id,
+			label,
+			name,
+			value,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			'minTouchHeight',
+			className
+		);
+
+		const elId = id || `${name}-${value}`;
+
+		return (
+			<Flex align='center'
+				className={classNames}
+				{...other}>
+				<FlexItem shrink>
+					<input	type='checkbox'
+						name={name}
+						value={value}
+						checked={this.state.checked}
+						id={elId}
+						onChange={this.onChange}
+					/>
+				</FlexItem>
+				<FlexItem>
+					<label className='label--minor' htmlFor={elId}>{label}</label>
+					{children}
+				</FlexItem>
+			</Flex>
+		);
+	}
+}
+
+Checkbox.propTypes = {
+	checked: React.PropTypes.bool,
+	label: React.PropTypes.string,
```

Comment:
there is technically [a way to get a raw translated string](https://github.com/meetup/mup-web/blob/master/docs/i18n.md#get-the-translated-string-as-a-value-in-js), but I agree that it would be nice to just pass in a `<FormattedMessage>` element.